sandesh/library/cpp/SConscript

```diff
@@ -11,9 +11,9 @@ env.CppDisableExceptions()
 
 env.Append(CPPPATH = [
     '#build/include',
-    '#controller/src',
```

Comment:
Ditto. And also, would like to understand why you had to move the entry for `Dir('.').abspath` ?